README.adoc

```diff
@@ -209,11 +204,7 @@ http://localhost:9080/LibertyProject/System/properties[^] URL.
 
 == Testing the service
 
-You can test this service manually by starting a server and pointing a web browser at the
-http://localhost:9080/LibertyProject/System/properties[^] URL. Automated tests are a much better
-approach because they trigger a failure if a change introduces a bug. JUnit and the JAX-RS Client
-API provide a simple environment to test the application.
-
+JUnit and the JAX-RS Client API provide a simple environment to test the application.
```

Comment:
Is there a reason to remove the text about testing it in a browser?  I agree with the text that automated tests like JUnit are superior, but it is often nice to be able to manually test out services - and the browser is a great way to do it.

Reply:
Addressed.